index.html

```diff
@@ -12,20 +12,31 @@
 <body>
     <header>
         <h1>Emilie Lokietz</h1>
+
         <nav>
             <a href="#kontakt">Kontakt</a>
             <a href="#lebenslauf">Biographie</a>
             <a href="">Mein Hund</a>
             <!-- da ist die Idee auf eine neue Website überzuleiten mit einer Galerie von den Fotos von meinem Hund-->
```

Comment:
In that case you would need a second HTML document with the name: `dogs.html` and there you can add the gallery 🙂 Maybe google something like: how to link two HTML pages